packages/cli/src/commands/hydrogen/dev.ts

```diff
@@ -125,6 +147,11 @@ async function runDev({
     },
     async onRebuildFinish() {
       console.timeEnd(LOG_REBUILT);
+
+      if (!miniOxygenStarted && (await serverBundleExists())) {
+        console.log(''); // New line
```

Comment:
I _think_ `ouput.newLine()` exists and could be used here.

Reply:
@cartogram I always try `output.newLine()` first, just to realize that it prints `undefined` in the console instead of an actual new line, so I always change it back to `console.log('')` :/